### DISCUSSION TITLE
Stream stats toggle

### DISCUSSION THREAD

**Discussion by SeanHub** [ChristopherKlay/StadiaEnhanced#232] [Ideas & Requests]
Hi, can the stream stats that appear to the left of the stadia panel be toggled off/on via a setting. They clutter up the clean UI a little and move wherever the focus is which is a tad distracting.

**Comment by ChristopherKlay** [comment]
Version `1.20.3` just went live and should fix the issue with the monitor adjusting it's position based on the controller selected element.

I'll add a option to toggle the entire monitor down the line (:

**Comment by SeanHub** [comment]
Awesome, thank you!
